monitors/rootmonitor/CMakeLists.txt

```diff
@@ -0,0 +1,83 @@
+if(WIN32)
+  message(STATUS "Online ROOT monitor is not to be built on windows")
+  return()
+endif()
+
+list(APPEND CMAKE_PREFIX_PATH $ENV{ROOTSYS})
+if(WIN32)
+  find_package(ROOT REQUIRED COMPONENTS libGui libCore libHist libRIO)
+else()
+  find_package(ROOT REQUIRED COMPONENTS Gui)
+endif()
```

Comment:
Maybe you could be a bit more consistent here: If you do not support windows at all, the if-else in 7-11 is not necessary. Otherwise, you can remove lines 1-4.

Reply:
If it is supposed to become the base for a new standard monitor to replace OnlineMon (eudaq1) / StdEvtMon (eudaq2) then one should try to get it running on all our standard supported platforms. What problem did you encounter with VS on Windows?

Reply:
@lhuth, @arummler 
Thanks for the review! I indeed forgot I disabled this on win systems (pretty much like this is done in the `StdEvtMon`, see [here](https://github.com/eudaq/eudaq/blob/master/monitors/onlinemon/CMakeLists.txt#L1-L4)), as I did not have a working environment to test these developments on.
I will try to set up a VM with Win/VS for further tests (and re-enable the compilation for the time being to see if the CI complains...)